test/teos/unit/test_api.py

```diff
@@ -461,13 +461,20 @@ def test_get_appointment_not_registered_user(client):
     test_get_random_appointment_registered_user(client, tmp_sk)
 
 
-def test_get_appointment_in_watcher(api, client, appointment):
+def test_get_appointment_in_watcher(client, api, appointment, monkeypatch):
+
     # Mock the appointment in the Watcher
     uuid = hash_160("{}{}".format(appointment.locator, user_id))
     extended_appointment_summary = {"locator": appointment.locator, "user_id": user_id}
     api.watcher.appointments[uuid] = extended_appointment_summary
     api.watcher.db_manager.store_watcher_appointment(uuid, appointment.to_dict())
 
+    def mock_authenticate_user(*args, **kwargs):
```

Comment:
Given this is defined twice, it could be a place before the tests as a non-internal method, just with the rest of the non-test functions

Reply:
Done in 3d016331a1a94d4eade971a6f1955c4ff23294dc.